docs/sources/reference/commandline/cli.md

```diff
@@ -1410,7 +1410,7 @@ for further details.
 
     rename a existing container to a NEW_NAME
 
-The `docker rename` command allows the container to be renamed to a different name.  
```

Comment:
It looks like this includes some changes for different areas than what this PR is about.  If this was attention can you revert these changes so that we have a single commit for one feature/area please?  It just makes things easier to review, merge, or work with the commit in the future.